src/_pytest/runner.py

```diff
@@ -220,12 +231,22 @@ def call_runtest_hook(item, when: "Literal['setup', 'call', 'teardown']", **kwds
 
 @attr.s(repr=False)
 class CallInfo:
-    """ Result/Exception info a function invocation. """
+    """ Result/Exception info a function invocation.
+
+    :param excinfo: `ExceptionInfo` describing the captured exception if any
```

Comment:
```suggestion
    :param excinfo: (Optional[ExceptionInfo]) The captured exception, if any.
```

Reply:
I'm not fond of using `typing` symbols in docstring, they are not for humans. Especially as the real official meaning of `Optional` is that the **`type`** is optional (hence the attribute is nonable).

Also, Sphinx might like the use of backticks, no ?

Reply:
The sphinx docs already display type annotations for annotated functions, for example: https://docs.pytest.org/en/latest/reference.html#pytest-exit. Normally with annotations we don't need to write types in comments, but attrs is a special case.

I would not say type annotations are not for humans -- programmers are humans too, at least for now :) I except a lot of Python programmers will eventually learn what they mean.

I like to have the `Optional`, it tells me immediately it might be `None`, no need to read the text.

I'm not sure about the backticks, I suggested to remove it because you didn't add it for the others. I'm fine with whatever works...

Reply:
oh thanks I forgot to include them for the others.